src/styles/utils/_variables.scss

```diff
@@ -0,0 +1,8 @@
+$week: 
+  "mon" 0px,
+  "tue" 101px,
+  "wed" 202px,
+  "thu" 303px,
+  "fri" 404px,
+  "sat" 505px,
+  "sun" 606px;
```

Comment:
split this info to different independent values for example 
`mon 1, tue 2...;
$gap: 1px;
$size: 100px;`